widgets/data-action-widgets/data-action-only/manifest.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "data-action-only",
+  "label": "Data Action Only",
+  "type": "widget",
+  "version": "1.16.0",
+  "exbVersion": "1.16.0",
+  "author": "Esri R&D Center Beijing",
+  "description": "This is a barebones, simple widget. It includes an example of how to use the config file, and it includes a placeholder SVG icon. It intentionally does not include the following so you can use this to start from scratch: i18n translations, a Settings panel, unit tests. See the documentation for more information on how to implement these features.",

```

Should this description be the same as what is in the readme?

```suggestion
  "description": "This sample demonstrates how to create a widget that has the data action only.",

```